### PR TITLE
Add a flag to select between nondecomposed and decomposed attention

### PR DESCRIPTION
**Pull request by KyleHerndon**

### CODE REVIEW

File: sharktank/sharktank/export_layer/export_paged_attention.py [hunk 1]
```diff
@@ -193,6 +193,12 @@ def main():
         action="store_true",
     )
 
+    parser.add_argument(
```
Comment (dan-garvey) [inline sharktank/sharktank/export_layer/export_paged_attention.py:196]: I think we need to move this to CLI to enable re-use with eager.

File: sharktank/sharktank/ops/default_impls.py [hunk 2]
```diff
@@ -405,7 +405,7 @@ def rms_norm_default(x, weight, *, epsilon: float) -> Tensor:
     variance = x.pow(2).mean(-1, keepdim=True)
     output = x * elementwise(torch.rsqrt, variance + epsilon)
     # The cast here is to match the hf implementation, affects numerics
-    output = weight * to(output, weight.dtype)
+    output = unbox_tensor(weight) * to(output, weight.dtype)
```
Comment (dan-garvey) [inline sharktank/sharktank/ops/default_impls.py:408]: I don't think this breaks anything but why do we need it?

**Review by dan-garvey — COMMENTED** [review]
minor comments

File: sharktank/sharktank/export_layer/export_paged_attention.py (hunk 1 above)
Comment (dan-garvey) [reply sharktank/sharktank/export_layer/export_paged_attention.py:196]: can leave this as a todo if you want

File: sharktank/sharktank/ops/default_impls.py (hunk 2 above)
Comment (KyleHerndon) [reply sharktank/sharktank/ops/default_impls.py:408]: When exporting, torch gets upset when the multiplication has one argument of a `sharktank` tensor and one of a `torch` tensor type. The `to` op always returns a torch tensor, so this fixes them both to be `torch` tensors. Almost all the functions in `default_impls.py` are calling `unbox_tensor` on their inputs. I'm not sure why this patch turned up this particular issue, but it was in fact failing before I added this.

Comment (KyleHerndon) [reply sharktank/sharktank/ops/default_impls.py:408]: Actually, using that fix breaks other things. Changing it to an explicit `ops.elementwise(torch.mul, x, y)` works better.

**Review by dan-garvey — APPROVED** [review]

**Pull request merged**